### PR TITLE
Biometric server

### PR DESCRIPTION
**Pull request by abhijith0505**

### CODE REVIEW

File: servers/biometric_signal_sensor_interface/src/health_monitor/monitor.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
+from __future__ import absolute_import, division, print_function
+import sys
+import os
+sys.path.insert(0, '../hexoskinHelper')
+sys.path.insert(0, '../anomaly_detector')
+import utilityHelper as util
```
Comment (mtambos) [inline servers/biometric_signal_sensor_interface/src/health_monitor/monitor.py]: wrong naming style

Comment (abhijith0505) [reply servers/biometric_signal_sensor_interface/src/health_monitor/monitor.py]: It should be changed to snake_case right?

**Comment by mtambos** [comment]
that's right

**Comment by abhijith0505** [comment]
Updated

**Pull request merged**